Sortable.js

```diff
@@ -447,6 +447,7 @@
 
 			if (activeGroup.pull == 'clone') {
 				cloneEl = dragEl.cloneNode(true);
+				cloneEl.shadowRoot.innerHTML = dragEl.shadowRoot.innerHTML; //copy shadow root for polymer templates
```

Comment:
`cloneEl.shadowRoot. innerHTML` — TypeError: Cannot read property 'innerHTML' of `null`